frontend/packages/dev-console/src/components/gitops/GitOpsEmptyState.tsx

```diff
@@ -16,11 +17,14 @@ const gitopsImage = () => (
   <img className="odc-gitops-empty-state__icon" src={gitopsIcon} alt="GitOps" />
 );
 
-const GitOpsEmptyState: React.FC<GitOpsEmptyStateProps> = ({ emptyStateMsg }) => (
-  <EmptyState variant={EmptyStateVariant.full}>
-    <EmptyStateIcon variant="container" component={gitopsImage} />
-    <EmptyStateBody>{emptyStateMsg}</EmptyStateBody>
-  </EmptyState>
-);
+const GitOpsEmptyState: React.FC<GitOpsEmptyStateProps> = ({ emptyStateMsg }) => {
+  const { t } = useTranslation();
+  return (
+    <EmptyState variant={EmptyStateVariant.full}>
+      <EmptyStateIcon variant="container" component={gitopsImage} />
+      <EmptyStateBody>{t('devconsole~{{emptyStateMsg}}', emptyStateMsg)}</EmptyStateBody>
```

Comment:
the emptyMsg string is being set in `GitopsDetailsPage` , how about translating the string there ?